experimental/InvariantTheory/test/runtests.jl

```diff
@@ -63,4 +63,40 @@
     RT = invariant_ring(r)
     A, _ = affine_algebra(RT)
     @test ngens(modulus(A)) == 1
+
+    #no reynolds operator required/available:
+
+    #SL(2) over symmetric forms of degree 2
+    g = linearly_reductive_group(:SL,2,QQ)
+    r = representation_on_forms(g,2)
+    M = representation_matrix(r)
+    ringg = parent(M[1,1])
+    z = gens(ringg)
+    f = z[1]*z[4] - z[2]*z[3] - 1
+    G = linearly_reductive_group(ideal([f]))
+    R = representation_reductive_group(G,M)
+    RG = invariant_ring(R)
+    F = fundamental_invariants(RG)
+    @test length(F) == 1
+    X = polynomial_ring(RG)
+    g = gens(X)
+    @test F[1] == g[1]*g[3] - g[2]^2
+
+    #SL(2) over symmetric forms of degree 4:
+    #Here we see that without the Reynolds operator we get many more generators.
+    g = linearly_reductive_group(:SL,2,QQ)
+    r = representation_on_forms(g,4)
+    #with reynolds operator we get 2 fundamental invariants:
+    rg = invariant_ring(r)
+    FF = fundamental_invariants(rg)
+    @test length(FF) == 2
+    M = representation_matrix(r)
+    ringg = parent(M[1,1])
+    z = gens(ringg)
+    f = z[1]*z[4] - z[2]*z[3] - 1
+    G = linearly_reductive_group(ideal([f]))
+    R = representation_reductive_group(G,M)
+    RG = invariant_ring(R)
+    F = fundamental_invariants(RG)
+    @test length(F) == 27
```

Comment:
- When the user constructs the group as SL(n), they are automatically using the Reynold's operator from Cayley's omega process to compute fundamental invariants. 
- When the user creates the group using only the defining ideal, there is no known Reynolds operator. Therefore the algorithm which does not require a Reynold's operator is used to compute fundamental invariants (or rather, just generators of the invariant ring?). 

This example shows the difference between the two algorithms. SL(2) over degree 4 polynomials has 2 fundamental invariants when using Cayley's omega process. Without a Reynolds operator, we get 27 generators.